src/witan/models/dem/ccm/components_functions.clj

```diff
@@ -78,3 +68,80 @@
                                vec
                                (sort-by (apply juxt col-key))
                                ds/dataset)))
+
+(defn lag
+  [dataset col-key]
+  (-> dataset
+      (ds/to-map)
+      (col-key)
+      (#(cons 0 %))
+      (butlast)))
+
+(defn calc-proportional-differences
```

Comment:
Asked Wil what those numbers actually are "The birth trend is a rate per 1,000 women. The death trend is a rate per 100,000 people."